frostsnap_coordinator/src/backup_run.rs

```diff
@@ -1,43 +1,40 @@
 use crate::persist::{Persist, ToStringWrapper};
 use bdk_chain::rusqlite_impl::migrate_schema;
-use frostsnap_core::{AccessStructureId, AccessStructureRef, DeviceId, Gist, KeyId};
+use frostsnap_core::{AccessStructureId, AccessStructureRef, Gist, KeyId};
 use rusqlite::params;
 use std::collections::BTreeMap;
-use std::time::{SystemTime, UNIX_EPOCH};
 use tracing::{event, Level};
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct DisplayBackupState {
+    pub confirmed: bool,
+    pub legacy_display_confirmed: bool,
+}
+
 #[derive(Default, Debug)]
 pub struct BackupState {
-    // Maps each access_structure_ref to a map of device_id -> timestamp.
-    runs: BTreeMap<AccessStructureRef, BTreeMap<DeviceId, Option<u32>>>,
```

Comment:
Even though we're removing the "too quick" feature, it could be useful to leave the confirmation as a timestamps that can be used to prompt a backup/health check in the future.

Though on second thought this would probably be an entirely separate feature, "connect device X or validate backup". If we did introduce health checks, we could set the time of the first Backup Run completion as the first timestamp, then prompt at t+N from there.

Reply:
This is a good point. I think perhaps health checks can just be signing something. Optionally you could check the backup as well. but I don't think that determines when you need to do the health check. Also signing a tx would reset the timer.

I think if we want a backup check timer then that also has to start when you check the backup rather than writing it down.